### ISSUE TITLE
Organization selector box with very low contrast on my setup

### ISSUE THREAD

**Issue by gbif-portal** [gbif/portal-feedback#753]
**Organization selector box with very low contrast on my setup**

Like similar such lists on following pages, the list of organizations that opens up from the "First, please see if..." dialogue has very low contrast in my Firefox/Windows/screen setup: The organization names themselves are in a grey shade that is difficult to read, and the text matches are in an even lighter shade. When working under less than ideal light conditions, it is possible to overlook that there is a list with text entries opening at all. (Happy to demonstrate if helpful; screenshot to follow).

-----
fbitem-ab3ccb02d409d7c1b6652ceee60e3937ce068b3e
User provided contact info: @ahahn-gbif
System: Firefox 57.0.0 / Windows 10 0.0.0
Referer: https://www.gbif.org/become-a-publisher
Window size: width 824 - height 950
[API log](http://elk.gbif.org:5601/app/kibana?#/discover?_g=(refreshInterval:(display:Off,pause:!f,value:0),time:(from:'2018-01-09T08:20:04.142Z',mode:absolute,to:'2018-01-09T08:26:04.142Z'))&_a=(columns:!(_source),index:'prod-varnish-*',interval:auto,query:(query_string:(analyze_wildcard:!t,query:'response:%3E499')),sort:!('@timestamp',desc)))
[Site log](http://elk.gbif.org:5601/app/kibana?#/discover?_g=(refreshInterval:(display:Off,pause:!f,value:0),time:(from:'2018-01-09T08:20:04.142Z',mode:absolute,to:'2018-01-09T08:26:04.142Z'))&_a=(columns:!(_source),index:'prod-portal-*',interval:auto,query:(query_string:(analyze_wildcard:!t,query:'response:%3E499')),sort:!('@timestamp',desc)))
System health at time of feedback: OPERATIONAL

**Comment by ahahn-gbif** [comment]
![becomeapublisher_selectorcontrast](https://user-images.githubusercontent.com/13435656/34711348-64f2e182-f51f-11e7-83ee-c6e7d4b8990f.PNG)

**Comment by MortenHofft** [comment]
Yes it is annoying. It is a default in the library that we use. Not sure how to customize it, but I agree that is is annoyingly faded
